maven-plugin-tools-api/src/test/java/org/apache/maven/tools/plugin/javadoc/JavadocReferenceTest.java

```diff
@@ -45,6 +45,8 @@ void testParse()
                       JavadocReference.parse( "package.Class#member(ArgType1,ArgType2) label" ) );
         assertEquals( new JavadocReference( Optional.of("my.module"), Optional.of( "package.Class" ), Optional.of( "member(ArgType1,ArgType2)" ), Optional.of("label") ),
                       JavadocReference.parse( "my.module/package.Class#member(ArgType1,ArgType2) label" ) );
+        assertEquals( new JavadocReference( Optional.empty(), Optional.empty(), Optional.of( "member" ), Optional.of("label with spaces") ),
+                JavadocReference.parse( "#member label with spaces" ) );
```

Comment:
My bad, but this id/anchor contains spaces...?!

Reply:
This is no anchor. It is a Javadoc code reference.